src/stores/auth.js

```diff
@@ -17,6 +17,7 @@ export const useAuthStore = defineStore('auth', {
     username: null,
     mustReauthenticate: false,
     isReturningUser: null,
+    redirectGameId: null,
```

Comment:
As I said above, I think this data will be better served as a url param so that we can recover it on page refresh